pkg/server/settingswatcher/settings_watcher_external_test.go

```diff
@@ -488,12 +489,13 @@ func TestStaleRowsDoNotCauseSettingsToRegress(t *testing.T) {
 		rows, err := s.DB().Scan(ctx, k, k.PrefixEnd(), 0 /* maxRows */)
 		require.NoError(t, err)
 		dec := settingswatcher.MakeRowDecoder(codec)
+		var alloc *tree.DatumAlloc
```

Comment:
If you wanted this to avoid re-allocating, you'd not have a `*` here and you'd take the address